language/tools/move-cli/README.md

```diff
@@ -75,45 +96,50 @@ a directory structure that mimics the Move memory model
 
 ### Project structure
 
-Each Move CLI project with a given `name` should have the following structure to
-it:
-
-```
-name/
-├── src # Directory containing all Move source modules
-│   ├ ...
-│   └── Module.move
-└── scripts # Directory containing all Move scripts
-    ├ ...
-    └── script.move
-```
-
-Let's now create a Move project that we'll use for the code in this README and `cd` into it:
+Each sandbox command is run in the context of a Move package. So let's create a
+Move package that we'll use for the code in this README and `cd` into it:
 
 ```shell
-$ mkdir readme
+$ move package new readme
 $ cd readme
-$ mkdir src
-$ mkdir scripts
 ```
 
 ### Compiling and running scripts
 
-Let's first start out with a simple script that prints its `signer`:
+Let's first start out with a simple script that prints its [`signer`](https://diem.github.io/move/signer.html):
 
 ```rust
 script {
```

Comment:
My personal preference is, when instructing the reader to create a file with certain contents, to include a comment indicating the name of the file in the code block:

```suggestion
// sources/debug_script.move
script {
```